media-sound/ncmpcpp/ncmpcpp-0.9.2_p20221231.ebuild

```
@@ -0,0 +1,71 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+
+NCMPCPP_COMMIT="9f44edf0b1d74da7cefbd498341d59bc52f6043f"
+
+DESCRIPTION="featureful ncurses based MPD client inspired by ncmpc"
```

Capital first letter:
```suggestion
DESCRIPTION="Featureful ncurses based MPD client inspired by ncmpc"
```